src/command/dev/router/runner.rs

```diff
@@ -32,9 +32,11 @@ pub struct RouterRunner {
     client_config: StudioClientConfig,
     plugin_exe: Option<Utf8PathBuf>,
     router_handle: Option<BackgroundTask>,
+    license: Option<Utf8PathBuf>,
 }
 
 impl RouterRunner {
+    #[allow(clippy::too_many_arguments)]
```

Comment:
I'm cheating here (again), but fixing this will require some refactoring that might complicate this PR. Happy to do it, but wondering if it's worth it given the potential rover dev rewrite?